package.json

```diff
@@ -23,7 +23,7 @@
     ]
   },
   "scripts": {
-    "test": "jest",
+    "test": "jest --coverage",
```

Comment:
We can add a new script to handle test coverage, like `test:coverage`.

Reply:
Ok.